README.md

```diff
@@ -1,13 +1,7 @@
-oysteR
-================
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-<p align="center">
-
-<img src="https://github.com/sonatype-nexus-community/oysteR/blob/master/docs/images/OysteR_LabeledInHex.png" width="350"/>
-
-</p>
+# oysteR <a href='https://tidyverse.tidyverse.org'><img src='man/figures/logo.png' align="right" height="138.5" /></a>
```

Comment:
I'm thinking maybe this was copypasta from a different readme? But I do like the suggestion of making the img ref relative.

```suggestion
# oysteR <a href='https://github.com/sonatype-nexus-community/oysteR'><img src='docs/images/OysteR_LabeledInHex.png' align="right" height="138.5" /></a>
```

Reply:
Good spot @brittanybelle - this is why we have code review ;)